docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3'
+services:
+  db:
+    image: postgres:9.6
+    restart: always
+    environment:
+      POSTGRES_USER: cachito
+      POSTGRES_PASSWORD: cachito
+      POSTGRES_DB: cachito
+      POSTGRES_INITDB_ARGS: "--auth='ident' --auth='trust'"
+
+  rabbitmq:
+    image: rabbitmq:3.7-management
+    environment:
+      RABBITMQ_DEFAULT_USER: cachito
+      RABBITMQ_DEFAULT_PASS: cachito
+    ports:
+      # The RabbitMQ management console
+      - 8081:15672
+
+  cachito-api:
+    build: .
+    command:
+    - /bin/sh
+    - -c
+    - >-
+      pip3 uninstall -y cachito &&
+      python3 setup.py develop --no-deps &&
+      cachito wait-for-db &&
+      cachito db upgrade &&
+      flask-3 run --reload --host 0.0.0.0 --port 8080
+    environment:
+      FLASK_ENV: development
+      FLASK_APP: cachito/web/wsgi.py
+    volumes:
+      - ./:/src:Z
```

Comment:
You want lower case z here. From `man docker run`:
> To change a label in the container context, you can add either of two suffixes :z or :Z to the volume mount.
These suffixes tell Docker to relabel file objects on the shared volumes. The z option tells Docker that two
containers share the volume content. As a result, Docker labels the content with a shared content label.
Shared volume labels allow all containers to read/write content.  The Z option tells Docker to label the
content with a private unshared label.  Only the current container can use a private volume.